packages/cli/src/commands/setup/sentry/sentry.js

```diff
@@ -0,0 +1,189 @@
+import fs from 'fs'
+import path from 'path'
+
+import { Listr } from 'listr2'
+import terminalLink from 'terminal-link'
+
+import {
+  addApiPackages,
+  addEnvVarTask,
+  addWebPackages,
+  colors as c,
+  getPaths,
+  isTypeScriptProject,
+  prettify,
+  writeFilesTask,
+} from '@redwoodjs/cli-helpers'
+import { errorTelemetry } from '@redwoodjs/telemetry'
+
+export const command = 'sentry'
+
+export const description = 'Support Sentry error and performance tracking'
+
+const PATHS = getPaths()
+
+export const builder = (yargs) => {
+  yargs
+    .option('force', {
+      alias: 'f',
+      default: false,
+      description: 'Overwrite existing sentry.js files',
+      type: 'boolean',
+    })
+    .epilogue(
+      `Also see the ${terminalLink(
+        'Redwood CLI Reference',
+        'http://localhost:3000/docs/cli-commands#setup-sentry'
+      )}`
+    )
+}
+
+export const handler = async ({ force }) => {
+  const extension = isTypeScriptProject ? 'ts' : 'js'
+
+  const notes = []
+
+  const tasks = new Listr([
+    addApiPackages(['@envelop/sentry', '@sentry/node', '@sentry/tracing']),
+    addWebPackages(['@sentry/react', '@sentry/tracing']),
```

Comment:
We probably want to pin these dependencies to their latest major versions just to save ourselves a few open issues in the future when sentry decides to upgrade